src/blocks/box/Box.types.ts

```diff
@@ -21,15 +21,15 @@ export type BoxResponsiveProps = {
   /* Sets flex-direction css property */
   flexDirection?: ResponsiveProp<CSSProperties['flexDirection']>;
   /* Sets gap between the elements */
-  gap?: ResponsiveProp<SpaceType>;
+  gap?: ResponsiveProp<BlocksSpaceType>;
```

Comment:
gap cant take input for all four sides, we should define a different type for it

Reply:
added a new type. will remove the previous type afterwards because that will break thing s right now.